src/cmap/background.rs

```diff
@@ -40,10 +37,12 @@ impl ConnectionPool {
 
         let mut i = 0;
         while i < connection_manager.checked_in_connections.len() {
-            if connection_manager.checked_in_connections[i].is_stale(connection_manager.generation) {
+            if connection_manager.checked_in_connections[i].is_stale(connection_manager.generation)
+            {
```

Comment:
this hanging brace is kind of weird. could we configure rustfmt to break on one of the dot-operators instead?

Reply:
Unfortunately I think we're already set to do that, and it's failing to; see the default of the option [here](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#control_brace_style)